book/graphs/other/ltg01.qmd

```diff
@@ -171,12 +177,13 @@ g51 <- g51 + stat_summary(
   fun.data = stat_mean_ci, geom = "errorbar",
   aes(group = 1, linetype = mean_text)
 )
-g51
+plot <- g51
+plot
 ```
 
 #### Adding Median
 
-```{r 2_2_add_median}
+```{r plot52, test = list(plot_v52 = "plot")}
 # Instead of a mean, the median could be more relevant. Add the median and 95% CI as follows.
 median_text <- "Median +/- 95% CI"
```

Comment:
Snapshot fails because the `.rds` file cannot access `median_text` which is internally defined.
```
! could not find function "median_text"
```